### PR TITLE
Status icons Component 🌈

### PR DESCRIPTION
**Pull request by nakhbari**
Issue: https://github.com/fastlane/ci/issues/698

icons and colors subject to change, but now that's super easy with this modular component!

#### Desktop Screenshot
image

#### Mobile Screenshot
image

### CODE REVIEW

**Comment by taquitos** [comment]
What do you think about using something more like a smily-face shrugging instead of the yellow smily? That way it better communicates that we don't know what the state is yet, because we're waiting for the build to be finished.

**Comment by nakhbari** [comment]
@taquitos The icon types and colors will change. @chemichelle is working on it. This is just to get the infra in.

**Comment by taquitos** [comment]
Hawt. Love it.

File: web/app/common/components/status-icon/status-icon.component.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<ng-container [ngSwitch]="status">
+  <mat-icon *ngSwitchCase="BuildStatus.SUCCESS" class="fci-status-icon-success">
+    sentiment_very_satisfied
+  </mat-icon>
+  <mat-icon *ngSwitchCase="BuildStatus.PENDING" class="fci-status-icon-pending">
+    sentiment_neutral
```
Comment (taquitos) [inline web/app/common/components/status-icon/status-icon.component.html]: `sentiment_unknown` might be better?

Comment (taquitos) [reply web/app/common/components/status-icon/status-icon.component.html]: ehh, maybe not. Yo no se

Comment (nakhbari) [reply web/app/common/components/status-icon/status-icon.component.html]: These are all the strings provide by the material icon set https://material.io/icons/

`sentiment_unknown` isn't a provide icon.

**Pull request merged**